modules/administration/pages/prometheus.adoc

```diff
@@ -210,8 +210,7 @@ image::monitoring_enable_services.png[scaledwidth=80%]
 image::monitoring_configure_formula.png[scaledwidth=80%]
 
 
-
-== Set up Visualization with Grafana
+== Visualization with Grafana
 
 The Grafana website contains dozens of dashboards uploaded by the community.
```

Comment:
Maybe keep this?

Reply:
I just removed the "set up" and the topic about installing the package